tests/features/common/actions/action-menu.action.js

```diff
@@ -0,0 +1,35 @@
+import actionMenu from '../components/action-menu.component'
+
+const action = {
+  openActionMenu: async function(driver, selector) {
+    let action_menu = actionMenu(selector.value)
+    await driver
+      .findElements(action_menu.options)
+      .then(elements => {
+        if (elements.length === 0) {
+          return driver.findElement(action_menu.open_button)
+        }
+      })
+      .then(element => {
+        if (element) {
+          element.click()
+        }
+      })
+  },
+  selectOptionInActionMenu: async function(driver, selector, option) {
+    let action_menu = actionMenu(selector.value)
+    await driver.findElements(action_menu.options).then(elements => {
+      if (elements.length > 0) {
+        for (let indx in elements) {
+          elements[indx].getText().then(txt => {
+            if (txt === option) {
+              elements[indx].click()
+            }
+          })
+        }
+      }
+    })
```

Comment:
Again, trying to avoid mixing `await` style with `then` style, also I think the `.length > 0` may be redundant because the `for` loop will just not start if the array is empty.
```suggestion
    const elements = await driver.findElements(action_menu.options)
    for (const element of elements) {
      const txt = await element.getText()
      if (txt === option) {
        element.click()
      }
    }
```

Reply:
After some investigation, I found that the working variant is next:
```suggestion
    await driver.findElements(action_menu.options).then(elements => {
      if (elements.length > 0) {
        for (let indx in elements) {
          elements[indx].getText().then(txt => {
            if (txt === option) {
              elements[indx].click()
            }
          })
        }
      }
    })
```
const action_menu = actionMenu(selector.value)
    const elements = await driver.findElements(action_menu.options)
    for (const element of elements) {
      element.getText().then(txt => {
        if (txt === option) {
          element.click()
        }
      })
    }

Reply:
After some investigation I found that next variant is working:
```suggestion
    const action_menu = actionMenu(selector.value)
    const elements = await driver.findElements(action_menu.options)
    for (const element of elements) {
      element.getText().then(txt => {
        if (txt === option) {
          element.click()
        }
      })
    }```